tests/languages/sparql/function_feature.test

```diff
@@ -13,17 +13,17 @@ foo.-:
 ----------------------------------------------------
 
 [
-    ["PrefixedName", "foo:bar"],
-    ["PrefixedName", "foo:"],
-    ["PrefixedName", ":bar"],
-    ["PrefixedName", "foo:ba1"],
-    ["PrefixedName", "foo:1ba"],
-    ["PrefixedName", "foo:bar"],
-    ["PrefixedName", ":bar"],
-    ["PrefixedName", "foo:bar%20bar"],
-    ["PrefixedName", "foo:bar.bar"],
-    ["PrefixedName", "foo-2:"],
-    ["PrefixedName", "foo.-:"]
+		["PrefixedName", "foo:bar"],
+		["PrefixedName", "foo:"],

```

Comment:
Please correct the indentation to one tab. (same for the other tests except `keyword_feature.test`)